website/docs/concepts/combining_providers.mdx

```diff
@@ -1,14 +1,14 @@
 ---
-title: Combining providers
+title: Combining Provider States
```

Comment:
I'm not sure what this change brings. Could you explain your reasoning?

Reply:
Because it is the state of the provider that is combined not the providers themselves. In other words it is not about combining a FutureProvider and StateProvider, but about combining the states of each of them.